bugbot/rules/close_intermittents.py

```diff
@@ -17,7 +17,7 @@ def description(self):
         return "Intermittent test failure bugs unchanged in 21 days"
 
     def get_max_actions(self):
-        limit_per_run = 140
+        limit_per_run = 2100
         limit_per_team = 42
```

Comment:
This also likely needs a temporary increase to process all bugs returned by the query.

Reply:
I'm thinking let's see how long it takes to process the backlog, and re-evaluate in a couple of weeks. We don't want to change too many bugs at once.